### PR TITLE
Registration page back

### PR DESCRIPTION
**Pull request by gemuzkm**

### CODE REVIEW

File: src/main/java/tms/instaclone/dao/DAO.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package tms.instaclone.dao;
+
+import tms.instaclone.entity.Entity;
+
+public interface DAO<T extends Entity> {
```
Comment (simonpirko) [inline src/main/java/tms/instaclone/dao/DAO.java:5]: Few operations

File: src/main/java/tms/instaclone/entity/Entity.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package tms.instaclone.entity;
+
+public abstract class Entity {
+    private Long id;
```
Comment (simonpirko) [inline src/main/java/tms/instaclone/entity/Entity.java]: Primitive?

File: src/main/java/tms/instaclone/dao/DAO.java (hunk 1 above)
Comment (SergeyAnshin) [reply src/main/java/tms/instaclone/dao/DAO.java:5]: What do you mean? What other operations are needed for user registration?

File: src/main/java/tms/instaclone/entity/Entity.java (hunk 2 above)
Comment (SergeyAnshin) [reply src/main/java/tms/instaclone/entity/Entity.java]: fixed

**Review by simonpirko — APPROVED** [review]

**Pull request merged**